### PR TITLE
Feature: beta account activity

### PR DESCRIPTION
**Pull request by luixo**
This is a version of #937 with more context added to each record.

Preview links:

https://frontend-pr-1006-3r1e.onrender.com/beta/accounts/multisender.app.near
https://frontend-pr-1006-3r1e.onrender.com/beta/accounts/qbit.poolv1.near
https://frontend-pr-1006-3r1e.onrender.com/beta/accounts/9755042d33e148f87265c3343b69ed902939f166.lockup.near
https://frontend-pr-1006-3r1e.onrender.com/beta/accounts/foundation.near
https://frontend-pr-1006-3r1e.onrender.com/beta/accounts/82f09db4a072c66f89e8e226234181bb53f5ec24.lockup.near

### CODE REVIEW

File: frontend/src/components/beta/accounts/AccountActivityBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      transfer: {
+        backgroundColor: "#F0FFEE",
+      },
+      refund: {
+        backgroundColor: "#F0FFEE",
+      },
+      restake: {
+        backgroundColor: "#EEFDFE",
+      },
+      "validator-reward": {
```
Comment (shelegdmitriy) [inline frontend/src/components/beta/accounts/AccountActivityBadge.tsx:37]: Why you don't use camelCase here?

File: frontend/src/components/beta/accounts/AccountActivityView.tsx [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+  const isDeltaAmountPositive = JSBI.greaterThan(deltaAmount, BI.zero);
+  const absoluteDeltaAmount = isDeltaAmountPositive
+    ? deltaAmount
+    : JSBI.multiply(deltaAmount, BI.minusOne);
+  const actions =
+    item.action.type === "batch" ? item.action.actions : [item.action];
+
+  return (
+    <>
+      {actions.map((subaction, subindex) => {
```
Comment (shelegdmitriy) [inline frontend/src/components/beta/accounts/AccountActivityView.tsx:216]: How about
```suggestion
      {actions.map((subAction, subIndex) => {
```
Way more readable I guess.

File: frontend/src/components/beta/accounts/AccountActivityView.tsx [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+                          : item.action.transactionHash
+                      }
+                    />
+                  </CopyWrapper>
+                </>
+              ) : null}
+            </TableElement>
+            <DateTableElement>
+              {subindex === 0
+                ? moment
```
Comment (shelegdmitriy) [inline frontend/src/components/beta/accounts/AccountActivityView.tsx:309]: I guess we don’t need `moment` here as we switched to another library

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

File: frontend/src/components/beta/accounts/AccountActivityBadge.tsx (hunk 1 above)
Comment (luixo) [reply frontend/src/components/beta/accounts/AccountActivityBadge.tsx]: No idea. I changed that to camelCase.

File: frontend/src/components/beta/accounts/AccountActivityView.tsx (hunk 3 above)
Comment (luixo) [reply frontend/src/components/beta/accounts/AccountActivityView.tsx]: That's what I found as well after rebase :)

**Review by shelegdmitriy — APPROVED** [review]

**Pull request merged**